versioned_docs/version-10.x/documentation.md

```
@@ -0,0 +1,106 @@
+    - [Mocking](/docs/10.x/mocking)
+- ## 패키지
+    - [Breeze](/docs/10.x/starter-kits#laravel-breeze)
+    - [Cashier (Stripe)](/docs/10.x/billing)
+    - [Cashier (Paddle)](/docs/10.x/cashier-paddle)
+    - [Dusk](/docs/10.x/dusk)
+    - [Envoy](/docs/10.x/envoy)
+    - [Fortify](/docs/10.x/fortify)
+    - [Folio](/docs/10.x/folio)
+    - [Homestead](/docs/10.x/homestead)
+    - [Horizon](/docs/10.x/horizon)
+    - [Jetstream](https://jetstream.laravel.com)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)